server_installation/topics/profiles.adoc

```diff
@@ -112,6 +105,13 @@ To enable a specific feature without enabling the full preview profile you can s
 bin/standalone.sh|bat -Dkeycloak.profile.feature.<feature name>=enabled`
 ----
 
+Or add to the `profile.properties file`:
+
```

Comment:
Please add "this" to the sentence, as in:
Or add this to the `profile.properties file`:

Reply:
@matthewhelmke Thank you, updated.